sljit_src/sljitNativePPC_common.c

```diff
@@ -653,6 +653,9 @@ SLJIT_API_FUNC_ATTRIBUTE void* sljit_generate_code(struct sljit_compiler *compil
 	compiler->error = SLJIT_ERR_COMPILED;
 	compiler->executable_offset = executable_offset;
 	compiler->executable_size = (sljit_uw)(code_ptr - code) * sizeof(sljit_ins);
+#if defined(SLJIT_INDIRECT_CALL) && SLJIT_INDIRECT_CALL
+	compiler->executable_size += (2 * sizeof(sljit_ins));
```

Comment:
There is an alignment code (`code_ptr++`) below.

Reply:
I am not sure how the trampoline is supposed to work for 64bit though (from a disassembly of a generated `void fn(void)` in AIX):

```
(dbx) p code.code
0x0a00000000000038 
(dbx) p compiler->executable_size
64 
(dbx) 0xa00000000000010/16i
0xa00000000000010 (???) 7c0802a6           mflr   r0
0xa00000000000014 (???) f821ff81           stdu   r1,-128(r1)
0xa00000000000018 (???) fbe10078            std   r31,0x78(r1)
0xa0000000000001c (???) f8010090            std   r0,0x90(r1)
0xa00000000000020 (???) 3be00000             li   r31,0x0
0xa00000000000024 (???) e8010090             ld   r0,0x90(r1)
0xa00000000000028 (???) ebe10078             ld   r31,0x78(r1)
0xa0000000000002c (???) 7c0803a6           mtlr   r0
0xa00000000000030 (???) 38210080           addi   r1,0x80(r1)
0xa00000000000034 (???) 4e800020            blr
0xa00000000000038 (???) 0a000000          tdlti   r0,0x0
0xa0000000000003c (???) 00000010       Invalid opcode.
0xa00000000000040 (???) 00000001       Invalid opcode.
0xa00000000000044 (???) 10001208        vmuleub   vr0,vr0,vr2
0xa00000000000048 (???) 00000000       Invalid opcode.
0xa0000000000004c (???) 00000000       Invalid opcode.
```